kolibri/core/tasks/management/commands/base.py

```diff
@@ -3,7 +3,8 @@
 
 from django.core.management.base import BaseCommand
 from iceqube.exceptions import UserCancelledError
-from tqdm import tqdm
```

Comment:
@aronasorman @lyw07 

Could we make this an optional dependency so that we don't need to monkey patch like this?

What's the appropriate place to file this issue?

Reply:
sorry I'm a bit confused.. why are we doing this here? @DXCanas

Reply:
if I recall, `tqdm` imports `multiprocessing`, which breaks things in Android @lyw07. This shim is a hacky way of fixing it.

Reply:
tqdm is not being used in iceqube, it is being used inside this module.

I think what would be best is just to do the try except on the import in here, and in the except import your mock tqdm object.